src/App.js

```diff
@@ -0,0 +1,30 @@
+/* eslint-disable import/extensions */
+import React, { useEffect } from 'react';
+import './App.css';
+import { useDispatch } from 'react-redux';
+import { Switch, Route, HashRouter as Router } from 'react-router-dom';
+import { MainPage } from './Components/MainPage';
+import { Details } from './Components/Details';
```

Comment:
- [ ] I am not able to run the app because of the error in these import statements. Kindly check to ensure that the files imports are correct. for instance, your `components` start with small `c` and not caps.